src/app/tests/TestEcosystemInformationCluster.cpp

```diff
@@ -49,11 +49,27 @@ const ClusterId kEcosystemInfoClusterId         = EcosystemInformation::Id;
 const AttributeId kDeviceDirectoryAttributeId   = EcosystemInformation::Attributes::DeviceDirectory::Id;
 const AttributeId kLocationDirectoryAttributeId = EcosystemInformation::Attributes::LocationDirectory::Id;
 
+class MockMatterContext : public MatterContext
```

Comment:
totally not a blocker, but given that this mirrors the valve context, do you think it's worthwhile having this be made generic and centralized somewhere? ie, should clusters define their OWN matter context, or should we have one matter context to rule them all with a standard test definition?

Reply:
If you are refering to MatterContext I agree that we should have a base class that takes care of the most common cases to avoid boilerplate.

If you are referring to MockMatterContext also wouldn't be a bad idea for a generic on, just there might be some differentiation between tests, but they can extend the mock matter context some more for their cases